chemprop/nn/predictors.py

```diff
@@ -230,7 +230,7 @@ class BinaryDirichletFFN(BinaryClassificationFFNBase):
 
     def forward(self, Z: Tensor) -> Tensor:
         Y = super().forward(Z)
-        alpha, beta = torch.chunk(Y, 2, 1)
+        alpha, beta = torch.unbind(Y, dim=-1)
```

Comment:
I don't think we can use `unbind` here because the output of `super().forward(Z)` has shape `b x (t*u)`. In MVEFFN and evidential we left this as chunk.

Reply:
Ah yes, you're right. It's alarming that our tests do not catch this. I will work on adding a test.

Reply:
As you're adding a test, a question on my mind is if `forward` of `BinaryDirichletFFN` should include a sigmoid so that it returns values between 0 and 1. While working on tests for the metrics, I ran into an issue with this. I think right now it is returning logits and should return probabilities. Also not sure about `MulticlassdirichletFFN`...

Reply:
(Also note that the CI fails for unrelated reasons as of yesterday. #1016)

Reply:
@hwpang are you able to take a look, and we can aim to merge this in this week?

Reply:
> As you're adding a test, a question on my mind is if `forward` of `BinaryDirichletFFN` should include a sigmoid so that it returns values between 0 and 1. While working on tests for the metrics, I ran into an issue with this. I think right now it is returning logits and should return probabilities. Also not sure about `MulticlassdirichletFFN`...

I think this perhaps should be a different PR so that we can get this PR in, which is blocking efforts for uncertainty modules. I think we need to add integration tests for `BinaryDirichletFFN` and `MulticlassdirichletFFN`. We also should add a CLI test for `MulticlassdirichletFFN`.